server/stream.go

```diff
@@ -771,7 +773,7 @@ func (mset *stream) setStreamAssignment(sa *streamAssignment) {
 	// Set our node.
 	mset.node = node
 	if mset.node != nil {
```

Comment:
Should this be `if mset.node != nil && peers != nil` or is it safe to set empty peers if `js` is `nil`?

Reply:
We did not check before, so ok to just set IMO.